nvme-print.c

```diff
@@ -4363,7 +4422,8 @@ static void nvme_show_plm_config(struct nvme_plm_config *plmcfg)
 
 static void nvme_show_iocs_vector(struct nvme_iocs_vector *iocs_vector)
 {
-	printf("\tNVM Command Set is selected: %s\n", iocs_vector->nvm ? "True":"False");
+	printf("\tNVM   Command Set is selected: %s\n", iocs_vector->nvm ? "True":"False");
+	printf("\tZoned Command Set is selected: %s\n", iocs_vector->zoned ? "True":"False");
```

Comment:
Zoned Namespace Command Set

Reply:
@MatiasBjorling thanks for reviewing. Didn't realize modified as zoned namespace command set :). will modify this.